00-HTML-CSS-basics/css/2-3/css/styles.css

```diff
@@ -0,0 +1,35 @@
+*[class$='r'] {
+	background-color: magenta;
+}
+
+*[class*='a']:not([class$='r']) {
+	background-color: blue;
+}
+
+html {
+	font-weight: 900;
```

Comment:
Not sure if this is the solution for exercise 2.3.6, but I think the practice refers to override a global value by adding more specificity (so you should try to override ``font-size`` defined on ``.header, .footer``, etc. selectors).